### PR TITLE
Broken Links Fixed

### PR DESCRIPTION
**Pull request by Rakesh9100**
## 👨‍💻 Changes proposed

1. The links in the section "Table of Contents" in the file Readme.md was broken. So fixed it.
2. Links were redirecting to one of the repository i.e. Chayan-11 and not the owner of the repo 'arpit456jain' so I fixed them all, and now they direct shows the following content on clicking the link.
3. One of the link that is "Project Setup" was showing Error page not found on opening, so I fixed it, and now it shows the required project setup steps.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by arpit456jain** [comment]
issue ?

**Comment by Rakesh9100** [comment]
> issue ?

What exactly are you asking ?

**Comment by KKhushhalR2405** [comment]
> > issue ?
> 
> What exactly are you asking ?

You must first open an [issue](https://github.com/arpit456jain/CodingEasy/issues) explaining how you want to contribute. Get assigned and then create a [PR](https://github.com/arpit456jain/CodingEasy/pulls).

**Comment by Rakesh9100** [comment]
> > > issue ?
> > 
> > 
> > What exactly are you asking ?
> 
> You must first open an [issue](https://github.com/arpit456jain/CodingEasy/issues) explaining how you want to contribute. Get assigned and then create a [PR](https://github.com/arpit456jain/CodingEasy/pulls).

Okay I understand. Can you and team see the PR this time please? Contribution from next time will be for sure by creating an issue.

File: README.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ This project is part of the [GirlScript Summer of Code 2020](https://www.gssoc.t
...
 
 ## 💻Table of Contents
-  - **[How to Contribute?](https://github.com/Chayan-11/CodingEasy/blob/master/README.md#how-to-contribute)**
-  - **[How to make a Pull Request?](https://github.com/Chayan-11/CodingEasy/blob/master/README.md#how-to-make-a-pull-request)**
-  - **[Project Setup](https://github.com/tandrimasingha/CodingEasy/blob/master/README.md#project-setup)**
-  - **[Project Admin](https://github.com/Chayan-11/CodingEasy/blob/master/README.md#project-admin)**
-  - **[Our Valuable Contributors](https://github.com/Chayan-11/CodingEasy/blob/master/README.md#our-valuable-contributors)**
+  - **[How to Contribute?](#👩‍💻-how-to-contribute)**
+  - **[How to make a Pull Request?](#📍-how-to-make-a-pull-request)**
+  - **[Project Setup](#⚙-project-setup)**
```
Comment (KKhushhalR2405) [inline README.md:48]: These 3 hyperlinks are not redirecting properly. Can you please check

**Review by KKhushhalR2405 — CHANGES_REQUESTED** [review]
Please have a look and do the needful.

**Review by KKhushhalR2405 — APPROVED** [review]
LGTM @arpit456jain

**Pull request merged**